### PR TITLE
Code Review Guidelines

### PR DESCRIPTION
**Pull request by cerealean**
code-review-guidelines.md 
* New code review guidelines added

.gitignore
 * Added for convenience

### CODE REVIEW

**Comment by elliottisonfire** [comment]
Post group review, should be good.

**Comment by cerealean** [comment]
@elliottisonfire it is in the base directory

**Comment by nnnnathann** [comment]
Might I suggest that the developer wishing to request a code review be available for an in person / gotomeeting demo of the changes at the start of the review, THEN we jump into reviewing the code and implementation details

I think this would help make the process more efficient

File: docs/development/code-review-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+###Tiny Features, Ticklers, and Bugs
+
+Code in this category should be reviewed by **at least** one other developer. To qualify for this category:
+- The card for code being reviewed is sized, at maximum, 2 points
+- A maximum of 15 lines of code were changed
+- The code being reviewed will only change a small area of the product
+
+###Small-Moderate Features, Ticklers, and Bugs
+
+Code in this category should be reviewed by **at least** two other developers. To qualify for this category:
```
Comment (elliottisonfire) [inline docs/development/code-review-guidelines.md]: Following @nnnnathann 's advice, maybe add a clause to the description along these lines
```
The developer **should** schedule a short demo or provide materials documenting proof of meeting acceptance criteria.
```

File: docs/development/code-review-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- The card for code being reviewed is sized no more than 5 points
+- The code being reviewed will not majorly impact the entire product
+
+###Moderate-Large Features
+
+Code in this category should be reviewed by **at least** three other developers. To qualify for this category:
```
Comment (elliottisonfire) [inline docs/development/code-review-guidelines.md]: Kind of same as above, just more prominent with requiring a demo.
```
The developer **must** schedule a demo and **should** also provide materials documenting proof of meeting acceptance criteria.
```

File: docs/development/code-review-guidelines.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+As a general rule, all code **must** be tested before being used on a production environment. Below are some guidelines 
+to follow when reviewing others or being reviewed yourself.
+
+##General Guidelines
+###Who Should Review
+
+Code should be reviewed by other developers who have significant technical experience. It is also preferred that code be 
+reviewed by other developers who have relevant knowledge of the product.
```
Comment (elliottisonfire) [inline docs/development/code-review-guidelines.md]: Hard line break?

File: docs/development/code-review-guidelines.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+- Have successfully-running unit tests, where applicable
+
+If the code reviewed does not meet this criteria, the developer should be consulted and the code review should not be 
+accepted until necessary changes are made. By accepting another developer's code, you are agreeing that, to the best of 
+your knowledge, the code being reviewed meets the above criteria.
+
+##How Many People Should Review My Code?
+
+For most code additions and changes, it is preferred that multiple people perform the code review. Some additional guidelines 
+are below:
```
Comment (elliottisonfire) [inline docs/development/code-review-guidelines.md]: Hard line break?

**Pull request merged**